atlassian/parse.go

```diff
@@ -0,0 +1,63 @@
+package atlassian
+
+import (
+	"github.com/amp-labs/connectors/common"
+	"github.com/amp-labs/connectors/common/jsonquery"
+	"github.com/spyzhov/ajson"
+	"strconv"
+)
+
+func getTotalSize(node *ajson.Node) (int64, error) {
+	return jsonquery.New(node).ArraySize("issues")
```

Comment:
I am not sure about general solution for response parsing.
There should be a dependency between Atlassian module and response format.

For now we make only 1 read call for "issues".